src/winetricks

```diff
@@ -15337,7 +15375,11 @@ _EOF_
     # CLSID path will get overwritten on prefix update. Setting ieproxy to
     # native doesn't help because setupapi ignores DLL overrides. To work
     # around this problem, copy native ieproxy to system32.
-    w_try cp -f "${W_PROGRAMS_X86_UNIX}/Internet Explorer/ieproxy.dll" "${W_SYSTEM32_DLLS}"
+    if [ "${W_ARCH}" = "win32" ]; then
+        w_try cp -f "${W_PROGRAMS_X86_UNIX}/Internet Explorer/ieproxy.dll" "${W_SYSTEM32_DLLS}"
+    else
+        w_try cp -f "${W_PROGRAMS_UNIX}/Internet Explorer/ieproxy.dll" "${W_SYSTEM64_DLLS}"
+    fi
```

Comment:
I'm not confident about my replacement of this ieproxy logic because I don't know the implications/background:
https://github.com/Winetricks/winetricks/commit/bd8c584691f66f93bb7e35dab1e4ca2dc5650867
https://bugs.winehq.org/show_bug.cgi?id=44005

Should it instead be?:
```bash
w_try cp -f "${W_PROGRAMS_X86_UNIX}/Internet Explorer/ieproxy.dll" "${W_SYSTEM32_DLLS}"
if [ "${W_ARCH}" = "win64" ]; then
    w_try cp -f "${W_PROGRAMS_UNIX}/Internet Explorer/ieproxy.dll" "${W_SYSTEM64_DLLS}"
fi
```